src/charts/helpers/auto-config.js

```diff
@@ -27,6 +27,11 @@ export function autoConfigure (config, cache, data) {
   if (config.height === "auto" && cache.container) {
     newConfig.height = cache.container && cache.container.clientHeight || 0
   }
+  if (config.useScrolling && cache.svgWrapper) {
```

Comment:
As you mentioned, I think autoconfig should just be for calculating any value to replace a configuration set to "auto" or to compute some values that are derived from the config. Maybe it should live right on chart.js build, like
```
cache.svgWrapper
      .style("flex", `0 0 ${config.chartWidth}px`)
      .style("height", `${config.height}px`)
      .style("overflow-x", config.useScrolling ? "scroll" : "hidden")
```

Reply:
thanks, that makes more sense, will update.